### ISSUE TITLE
腾讯云的gpu服务器为什么开了deepspeed比不开花的时间还要多

### ISSUE THREAD

**Issue by hi-pengyu** [InternLM/xtuner#651]
腾讯云的gpu服务器为什么开了deepspeed比不开花的时间还要多，开了加速

05/07 07:05:41 - mmengine - INFO - Checkpoints will be saved to /root/train/work_dirs/internlm2_chat_7b_qlora_oasst1_e3_copy.
/root/miniconda3/envs/xtuner-env/lib/python3.10/site-packages/torch/utils/checkpoint.py:464: UserWarning: torch.utils.checkpoint: the use_reentrant parameter should be passed explicitly. In version 2.4 we will raise an exception if use_reentrant is not passed. use_reentrant=False is recommended, but if you need to preserve the current default behavior, you can pass use_reentrant=True. Refer to docs for more details on the differences between the two variants.
  warnings.warn(

这个地方会卡8分钟

05/07 07:12:16 - mmengine - INFO - Iter(train) [  10/8364]  lr: 7.2308e-06  eta: 3 days, 19:47:10  time: 39.5536  data_time: 0.0088  memory: 9828  loss: 1.6036

不开加速的情况需

05/07 07:18:23 - mmengine - INFO - Iter(train) [  10/8360]  lr: 7.2308e-06  eta: 17:32:09  time: 7.5604  data_time: 0.0085  memory: 11648  loss: 1.5721

**Comment by LZHgrla** [comment]
@hi-pengyu 
请问是什么GPU，训练命令是什么样的，以及time在后续的iteration中是否会降低？

**Comment by hi-pengyu** [comment]
> @hi-pengyu 
> 请问是什么GPU，训练命令是什么样的，以及time在后续的iteration中是否会降低？

腾讯云t4显卡  默认配置基本啥也没动，卡的太严重了，time没看

换了阿里云v100 没这个问题，应该是显卡的问题
